README.md

```diff
@@ -36,8 +36,26 @@ repos.
 
 ## Usage
 
-TODO: fill in details and examples here.
+### Commitlint
 
+* Add a file called commitlint.config.js to your project root with the following content:
+
+```javascript
+'use strict';
+
+module.exports = {
+   extends: [ '@silvermine/eslint-config/commitlint.js' ],
```

Comment:
We will move this commitlint config from eslint-config-silvermine to this repo.

Can you copy that file from there to here and update this PR? This instructions will need to be updated too.

Reply:
Eventually we will need a corresponding PR to eslint-config-silvermine that removes the config from there and sets it up to use the config from this repo.

Reply:
Jeremy initialized the repo with that file already inside. Docs are updated!